core/App/ehsm_napi.cpp

```diff
@@ -1549,11 +1550,11 @@ char* NAPI_GenerateQuote(const char *challenge)
  *          }
  *      }
  */
-char* NAPI_VerifyQuote(const char *quote_base64, const char *nonce)
+char* NAPI_VerifyQuote(const char *quote_base64, const char *mr_signer_base64, const char *mr_enclave_base64, const char *nonce_base64)
 {
     RetJsonObj retJsonObj;
 
-    if (quote_base64 == NULL || nonce == NULL) {
+    if (quote_base64 == NULL || nonce_base64 == NULL || mr_signer_base64 == NULL || mr_enclave_base64 == NULL) {
```

Comment:
mrenclave/mrsigner could be null, if the input is NULL, it will expected to be failed in the comparation logic .

Reply:
copy understood